File: sky-aac/src/main/java/sk/SKHttpException.java
package sk;

/**
 * @author sky
 * @version 版本
 */
public class SKHttpException extends RuntimeException {

    public SKHttpException() {}

    public SKHttpException(String detailMessage) {
        super(detailMessage);
    }

    public SKHttpException(String message, Throwable cause) {
        super(message, cause);
    }

    public SKHttpException(Throwable cause) {
        super((cause == null ? null : cause.toString()), cause);
    }
}

File: sky-kotlin/src/main/java/sky/kotlin/core/exception/SKHttpException.kt
package sky.kotlin.core.exception

/**
 * @author sky
 * @version 版本
 */
class SKHttpException : RuntimeException {

    constructor() {}

    constructor(detailMessage: String) : super(detailMessage) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(cause: Throwable?) : super(cause?.toString(), cause) {}
}